### DISCUSSION TITLE
Android support

### DISCUSSION THREAD

**Discussion by v74863** [ZerBea/hcxdumptool#385] [Q&A] [answered]
Contradictory statements in README.md about Android support.

At first
`Unsupported: Big endian systems, Windows OS, macOS, Android, emulators or wrappers!`

But a little lower
`Compile for Android`
`Install Android NDK on your system and add it to PATH`
...
`Copy it to your phone and enjoy`

What do you mean? How we should interpret this?

**Comment by ZerBea** [comment, marked as answer]
Only as a help. (Instruction to compile the code)
If there is a special feature request regarding Android only I'll disregard it.

By latest commit I added:
Please note that Android is unsupported and feature requests are rejected.

**Comment by ZerBea** [comment]
Please take a look at README.md
Now it should be clear.
